include/rewd_controllers/helpers.hpp

```diff
@@ -47,6 +47,10 @@ std::vector<JointParameter> loadJointsFromParameter(
   ros::NodeHandle& nodeHandle, const std::string& jointsParameter,
   const std::string& defaultType);
 
+std::unordered_map<std::string, double> loadGoalConstraintsFromParameter(
+  ros::NodeHandle& nodeHandle,
+  const std::vector<JointParameter>& jointParameters);
```

Comment:
Nit: This function should be const since it doesn't change the internal state of this class.

Reply:
agree with the sentiment, but this function is not part of any class :)